packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -994,12 +986,14 @@ class SlackBridge {
 			let data = {
 				token: this.apiToken,
 				ts: this.getSlackTS(rocketMessage),
-				channel: this.slackChannelMap[rocketMessage.rid].id
+				channel: this.slackChannelMap[rocketMessage.rid].id,
+				as_user: true
```

Comment:
Added the Slack 'as_user' optional flag to get messages deleted by the user instead of the bot, but doesn't work in all cases as expected.  May submit a slack defect.